internal/gen-valuewrapper/main.go

```diff
@@ -99,7 +99,10 @@ func run(args []string) error {
 	return err
 }
 
-var _tmpl = template.Must(template.New("int.go").Parse(`// Copyright (c) 2020 Uber Technologies, Inc.
+var _tmpl = template.Must(template.New("int.go").Parse(`// Code generated by gen-valuewrapper.
+// @generated
```

Comment:
I've used `@generated Code generated by foo` in the past, not sure if `@generated` has to be on the first line, but might also be cleaner to just have it all on one line